### ISSUE TITLE
Uncaught (in promise) ReferenceError: ComlinkWorker is not defined

### ISSUE THREAD

**Issue by neslinesli93** [mathe42/vite-plugin-comlink#82]
Hi, I just found out about this plugin - amazing work!

I tried to integrate it in an app of mine, but to no avail, as it fails with the folllowing stacktrace:

```
App.tsx:10 Uncaught (in promise) ReferenceError: ComlinkWorker is not defined
    at initWorker (App.tsx:10:7)
    at App.tsx:21:5
    at commitHookEffectListMount (react-dom.development.js:23150:26)
    at commitPassiveMountOnFiber (react-dom.development.js:24926:13)
    at commitPassiveMountEffects_complete (react-dom.development.js:24891:9)
    at commitPassiveMountEffects_begin (react-dom.development.js:24878:7)
    at commitPassiveMountEffects (react-dom.development.js:24866:3)
    at flushPassiveEffectsImpl (react-dom.development.js:27039:3)
    at flushPassiveEffects (react-dom.development.js:26984:14)
    at react-dom.development.js:26769:9
```

I created a repo to reproduce the issue: https://github.com/neslinesli93/comlink-vite-reproducible. And [here is the code](https://github.com/neslinesli93/comlink-vite-reproducible/blob/master/src/App.tsx#L10) that instantiates a worker.

These are the steps I followed to create the repo from scratch, it's the `react-ts` template of `create-vite`:

```
yarn create vite comlink-vite-reproducible --template react-ts
cd comlink-vite-reproducible
yarn
yarn add -D vite-plugin-comlink
yarn add comlink
yarn dev
```

Thank you for your time :)

**Comment by mathe42** [comment]
Thanks for reporting! Found the bug will fix it! Expect a release soon!

**Comment by mathe42** [comment]
Should be fixed in 3.0.5 (released just know)
